ci-operator/step-registry/openstack/deprovision/bastionproxy/openstack-deprovision-bastionproxy-commands.sh

```diff
@@ -13,11 +13,6 @@ export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml
 CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
 PROXY_PORT_ID=$(<"${SHARED_DIR}"/PROXY_PORT_ID)
 
-if [[ -f "${SHARED_DIR}/squid-credentials.txt" ]]; then
```

Comment:
why is this being removed?

Reply:
because now mecha has a permanent proxy AND can host proxy jobs, so we don't want to skip that step registry.
on L23, we have a check if a bastion FIP exists, then we clear things out.